resource/abstractions/individual_access.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Abstractions
+---
+
+# Individual Access
+
+General Resource Profile Administration functions for checking an Individual User's access to a General Resource Profile. 
+{: .fs-6 .fw-300 }
+
+## `ResourceAdmin.get_my_access()`
+
+```python
+def get_my_access(self, resource: str, class_name: str) -> Union[str, bytes, None]:
+```
+
+#### 📄 Description
+
+Get the **Running User's** access to a general resource profile.
+
+#### 📥 Parameters
+* `resource`<br>
+  The **general resource profile** to get the current user's access level for.
```

Comment:
Change `current user's` to `running user's`

Reply:
changed